mariner/server/api.py

```diff
@@ -123,9 +123,10 @@ def list_files() -> str:
             if dir_entry.is_file():
                 sliced_model_file: Optional[SlicedModelFile] = None
                 if get_file_extension(dir_entry.name) in get_supported_extensions():
-                    sliced_model_file = read_cached_sliced_model_file(
-                        path / dir_entry.name
-                    )
+                    if b"Mac OS X" not in open(dir_entry, "rb").read(32):
```

Comment:
is there any way to verify if it's a resource fork file by checking the filename?

I'm a bit hesitant to land this as is since this will introduce a lot of I/O for people who have lots of files on their upload path... the Raspberry Pi is notoriously bad with I/O so this could turn out to be slow over time :/